engine/kvdocument/core/src/main/java/com/torodb/kvdocument/values/KvDecimal128.java

```diff
@@ -276,18 +273,20 @@ private BigDecimal clampAndRound(final BigDecimal initialValue) {
               "Exponent is out of range for Decimal128 encoding of " + initialValue);
         } else {
           BigInteger multiplier = BIG_INT_TEN.pow(diff);
-          value = new BigDecimal(initialValue.unscaledValue().multiply(multiplier),
-              initialValue.scale() + diff);
+          value =
```

Comment:
This seems ugly, it is not necessary to break the line just after the =. Change it to _value = new BigDecimal(_